### PR TITLE
Feature/book crud

### PR DESCRIPTION
**Pull request by ahmedtimur**
new Book CRUD branch

### CODE REVIEW

File: src/main/java/kg/eBook/ebookb5/services/book/AudioBookService.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    }
+
+    public void updateBook(Authentication authentication, Long bookId, AudioBookSaveRequest audioBook) {
+
+        User user = userRepository.findByEmail(authentication.getName()).get();
+        Book book = bookRepository.findById(bookId).orElseThrow(
+                () -> new NotFoundException("Книга с  ID: " + bookId + " не найдена"));
+
+        if (user.getRole().equals(Role.ADMIN) || user.getRole().equals(Role.VENDOR) && book.getTypeOfBook().equals(AUDIO_BOOK)) {
```
Comment (beksultancs) [inline src/main/java/kg/eBook/ebookb5/services/book/AudioBookService.java:80]: You Put    @PreAuthorize("hasAnyAuthority('VENDOR', 'ADMIN')") this annotation in api
it means user 100% is Vendor or Admin
you can remove this statement!

**Review by beksultancs — COMMENTED** [review]
left comments!

**Comment by beksultancs** [comment]
You have some conflicts with main branch*
pull before pushing changes

File: src/main/java/kg/eBook/ebookb5/dto/requests/UserRegisterRequest.java [hunk 2]
```diff
@@ -4,6 +4,8 @@
 import lombok.Getter;
 import lombok.Setter;
 
+import java.time.LocalTime;
+
```
[the remark this answers is not in the document]
Comment (ahmedtimur) [reply src/main/java/kg/eBook/ebookb5/dto/requests/UserRegisterRequest.java:8]: Sir why we need role fiels here?

**Comment by ahmedtimur** [comment]
I made pull, also little changes in other classes, maybe because of this conflict appears.

**Pull request merged**